addons/video/src/presenter.js

```diff
@@ -10,6 +10,8 @@ function Addonvideo_create() {
     var deferredSyncQueue = window.DecoratorUtils.DeferredSyncQueue(deferredQueueDecoratorChecker);
     var currentTime;
 
+    var escapedSeparator = '&&separator&&';
```

Comment:
moze lepiej const ?

Reply:
Jest to na tyle drobne, że wydaje mi się że można by tą zmianę wprowadzić jeżeli będą jeszcze potrzebne poprawki do ticketu., zamiast niepotrzebnie spowalniać cały proces.